src/protocols/two_party_ecdsa/lindell_2017/party_one.rs

```diff
@@ -251,7 +254,14 @@ impl Signature {
         let s_tag_tag = BigInt::mod_mul(&k1_inv, &s_tag.0, &temp.q());
         let s = cmp::min(s_tag_tag.clone(), &temp.q().clone() - s_tag_tag.clone());
 
-        Signature { s, r: rx }
+        let is_ry_odd = ry.tstbit(0);
```

Comment:
Can you add a comment to the code describing how recid is calculated and its purpose?

Reply:
Added comment.